org-formation/050-costs/cur.yaml

```diff
@@ -0,0 +1,17 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  curBucket:
+    Type: String
+Resources:
+  MonthlyCostReport:
+    Type: "AWS::CUR::ReportDefinition"
+    Properties:
+      ReportName: !Sub ""
```

Comment:
`!Sub ""` will evaluate to empty.

Reply:
Whoops, I meant to put simple and unique value here.